src/default_tests.jsonc

```diff
@@ -23,10 +23,10 @@
       "LASTYR" : 1981,
       "pod_list": [
           // Optional: PODs to run for this model only (defaults to all)
-          "Wheeler_Kiladis",
-          "EOF_500hPa",
-          "MJO_suite",
-          "MJO_teleconnection"
+          // "Wheeler_Kiladis",
```

Comment:
uncomment the first 4 PODs so that the default.jsonc file is in its original develop branch state